src/main/java/com/agh/introwertycznelosie/data/Major.java

```diff
@@ -0,0 +1,101 @@
+package com.agh.introwertycznelosie.data;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+public class Major {
+
+  @Id
+  @GeneratedValue
+  private int id;
+
+  private Faculties faculty;
+  private String fullName;
+  private String shortName; // czy to jest w ogóle potrzebne? Że np informatyka to będzie inf?
```

Comment:
Wydaje mi się, że to bardziej w stylu jak jest elektronika i telekomunikacja to moze ma skrot eit?

Reply:
No tylko to jest skrót od nazwy kierunku, a nie wydziału. Więc ja bym to wyrzuciła całkiem